src/test/java/ikysil/training/dao/OrderDaoTest.java

```diff
@@ -0,0 +1,49 @@
+package ikysil.training.dao;
+
+import ikysil.training.dao.impl.InMemoryOrderDao;
+import ikysil.training.ws.api.v1.dto.OrderDto;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class OrderDaoTest {
+
+    private OrderDao dao = new InMemoryOrderDao();
+
+    private OrderDto order1 = new OrderDto("1144", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("3257", "twenty two", "none", "4024", "seconduser@gmail.com");
+
+    @Before
+    public void before(){
+        dao.saveOrder(order1);
+        dao.saveOrder(order2);
+    }
+
+    @Test
+    public void test(){
+        OrderDto testOrder1 = dao.findOrder("1144");
+        assertEquals(order1, testOrder1);
+
+        OrderDto testOrder2 = dao.findOrder("3257");
+        assertEquals(order2, testOrder2);
+
+        OrderDto newOrder = new OrderDto("3257","thirty three", "noneone", "2440", "newuser@gmail.com");
+        dao.updateOrder("3257", newOrder);
+
+        List<OrderDto> orders = dao.findAll();
+        System.out.println(orders);
+        assertTrue(orders.contains(newOrder));
+    }
+
+
+    @After
+    public void after(){
+        dao.deleteOrder("1144");
+        dao.deleteOrder("3257");
```

Comment:
you do not need to clean after test as `OrderDaoTest` would be instantiated for each test and `new InMemoryOrderDao()` would be clean for each test.